### DISCUSSION TITLE
Is there a way to reset/rebuild just the "details" table in the database?

### DISCUSSION THREAD

**Discussion by GuyWhoDidThatThing** [photoprism/photoprism#4081] [Q&A] [answered]
Wound up with a corrupted DB and tracked the cause down to the `details` table in the DB. I dropped and re-created it to get out of the crash loop. Now I can see and navigate all of my photos, but as you'd expect none of their details work and lead to "Missing entity". Looking through the available `photoprism` commands didn't reveal an obvious answer to rebuild just this table. Hoping I don't have to start from scratch if I can help it. Any ideas?

**Comment by lastzero** [comment, marked as answer]
In theory, you can just delete it, restart PhotoPrism so that it gets automatically recreated, and then reindex your library.

**Reply by GuyWhoDidThatThing** [reply]
This ended up causing a boot loop with no discernible errors in the logs. I'm going to go ahead and do a full reset since I didn't have too much setup, but thank you for your time!
